packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -112,11 +156,32 @@ export const EnlargedDashboardItem = () => {
 
   const { type } = currentDashboardItem || {};
 
+  // If the report is a drilldown, we want to show a back button to the parent dashboard item
+  const getBackLink = () => {
```

Comment:
I was trying to think of some way to split out the drill down code as it makes this component quite confusing and it's in danger of becoming a scary component to touch (the OG component in the OG site is terrifying). One quick win would be to make a BackButton component in this file and move this back link logic into it.